code/obj/item/material.dm

```diff
@@ -494,14 +495,17 @@
 	name = "scrap"
 	desc = "Some twisted and ruined metal. It could probably be smelted down into something more useful."
 	icon_state = "scrap"
+	stack_type = /obj/item/raw_material/scrap_metal
 	burn_possible = 0
+	set_name = TRUE
 
 	New()
 		..()
 		icon_state += "[rand(1,5)]"
-
-/obj/item/raw_material/scrap_metal/steel
-	material_name = "Steel"
+		if(!src.material)
+			src.setMaterial(getMaterial("steel"))
+	steel
+		material_name = "Steel"
```

Comment:
why not just use `material_name = "Steel'` for the parent as well?

Reply:
like just adding that instead of the if statement and setmaterial? or just using that over setmaterial?
I wasn't sure the ideal way to do it so I just copied one way since I was tired 😎